### PR TITLE
[Twig] Update templates.rst: Adding installation command

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Looks like this isn't included anywhere...

### CODE REVIEW

File: templates.rst [hunk 1]
```diff
@@ -15,6 +15,12 @@ Twig: a flexible, fast, and secure template engine.
 Twig Templating Language
 ------------------------
 
+To intsall Twig, run:
```
Comment (MrYamous) [inline templates.rst]:
```suggestion
To install Twig, run:
```

File: templates.rst [hunk 2]
```diff
@@ -15,6 +15,12 @@ Twig: a flexible, fast, and secure template engine.
...
+
+.. code-block:: bash
```
Comment (MrYamous) [inline templates.rst]:
```suggestion
.. code-block:: terminal
```

**Comment by javiereguiluz** [comment]
Thanks Thomas!

While merging, I reworded this a bit to follow the same `Installation` section format used in the rest of docs (see https://github.com/symfony/symfony-docs/commit/1d8d4b6c391a7a846574522654004a20907e9431)

**Comment by javiereguiluz** [comment]
I'm closing this PR manually because GitHub didn't close it.

This PR was merged in https://github.com/symfony/symfony-docs/commit/f1ff3ca22fdf0a18bb53ed31538b15b1a4b9cddc

**Pull request closed without merging**